src/meta/src/hummock/compaction/compaction_config.rs

```diff
@@ -70,10 +73,26 @@ impl CompactionConfigBuilder {
     }
 
     pub fn build(self) -> CompactionConfig {
+        if let Err(reason) = validate_compaction_config(&self.config) {
+            tracing::warn!("Bad compaction config: {}", reason);
+        }
         self.config
     }
 }
 
+/// Returns Ok if `config` is valid,
+/// or the reason why it's invalid.
+pub fn validate_compaction_config(config: &CompactionConfig) -> Result<(), String> {
```

Comment:
sub_level_number_threshold_min = 1 is really small? We need a more restrictive value (DEFAULT_LEVEL0_STOP_WRITE_THRESHOLD_SUB_LEVEL_MIN)

Reply:
This validation rejects config that is absolutely illegal. I cannot tell a proper value for DEFAULT_LEVEL0_STOP_WRITE_THRESHOLD_SUB_LEVEL_MIN now.